src/main/java/com/sigma/cinematicketpro/service/AppUserService.java

```diff
@@ -0,0 +1,6 @@
+package com.sigma.cinematicketpro.service;
+
+import org.springframework.security.core.userdetails.UserDetailsService;
+
+public interface AppUserService extends UserDetailsService{
```

Comment:
bad naming AppUserService it will better to make it UserService etc

Reply:
fixed: AppUserService -> CtpUserService